src/darwin/Framework/CHIP/MTRDeviceController+XPC.h

```diff
@@ -147,7 +157,7 @@ NS_ASSUME_NONNULL_BEGIN
                     maxInterval:(NSNumber *)maxInterval
                          params:(NSDictionary<NSString *, id> * _Nullable)params
                     shouldCache:(BOOL)shouldCache
-                     completion:(void (^)(NSError * _Nullable error))completion;
+                     completion:(StatusCompletion)completion;
```

Comment:
Not sure if this PR is the right place to address this, but the 3 typedefs at the top of `MTRCluster.h` (including `StatusCompletion`) are missing the `MTR` prefix.

Reply:
That gets addressed in a PR that's still coming up, as I reland these things.